package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "babel-preset-kensho",
-  "version": "4.1.1",
+  "version": "4.2.0",
```

I prefer to do release commits e.g. 6785ace936736f10c9ebc1d40877ac09f4c65f3f. Open to rethinking this but let's keep it consistent for now.

```suggestion
  "version": "4.1.1",
```